<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="solderButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Военнослужащие</string>
    </property>
   </widget>
   <widget class="QPushButton" name="polkButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Полки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mobButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Мобилизация</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
